imap_processing/hit/l2/hit_l2.py

```diff
@@ -102,10 +112,11 @@ def hit_l2(dependency: xr.Dataset) -> list[xr.Dataset]:
 class IntensityFactors(NamedTuple):
     """A namedtuple to store factors for the intensity equation."""
 
-    delta_e_factor: np.ndarray
+    delta_e: np.ndarray
     geometry_factor: np.ndarray
     efficiency: np.ndarray
     b: np.ndarray
+    seconds: int
```

Comment:
seconds is the unit, not the variable, so I'd just making this more descriptive with "time" and then adding a comment about it needing to be in units of seconds.
```suggestion
    integration_time: int
```

Reply:
Good suggestion!